bps/templates/bps_ptc.yaml

```diff
@@ -18,5 +18,5 @@ payload:
   payloadName: "${INSTRUMENT}_${TICKET}_ptc"
   output: "${USER_CALIB_PREFIX}${INSTRUMENT}/calib/${TICKET}/${TAG}/ptcGen.${RERUN}"
   butlerConfig: "${REPO}"
-  inCollection: "${USER_CALIB_PREFIX}${INSTRUMENT}/calib/${TICKET},${RAW_COLLECTION},${CALIB_COLLECTIONS}"
+  inCollection: "${RAW_COLLECTION},${CALIB_COLLECTIONS}"  # ${USER_CALIB_PREFIX}${INSTRUMENT}/calib/${TICKET}
```

Comment:
We need to discuss the right way of doing a bps run with these templates that doesn't involve doing the full stack.  (I have a hack that I don't love).  But please revert this change in the meantime.